share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -434,8 +436,8 @@ e4s-rocm-external-build:
     SPACK_CI_STACK_NAME: e4s-oneapi
 
 e4s-oneapi-generate:
-  extends: [ ".e4s-oneapi", ".generate-x86_64"]
-  image:  ghcr.io/spack/ubuntu22.04-runner-amd64-oneapi-2024.0.0:2024.01.16b
+  extends: [ ".e4s-oneapi", ".generate"]
+  image: ecpe4s/ubuntu20.04-runner-x86_64-oneapi:2023.07.21
```

Comment:
Let's not change the image tags for other stacks?  Probably an artifact of the rebase, but needs to be fixed.